src/utils/index.ts

```diff
@@ -236,3 +236,17 @@ export const getSvgForGoalPps = (count: number) => {
       return "ThreeAvatars";
   }
 };
+
+export const formatHoursText = (hours: string | null) => {
+  if (hours === null) {
+    return "";
+  }
+  if (typeof hours === "string") {
```

Comment:
remove this check as you have already validated the hours variable is not null

Reply:
`const formatHoursText = (hours: string | null) => {`
had to check for null due to the type of parameter being either string or null

Reply:
no i m talking about   if (typeof hours === "string") {
this check